commercial/app/views/jobs/jobsV2.scala.html

```diff
@@ -5,6 +5,8 @@
   omnitureId: Option[String],
   clickMacro: Option[String])(implicit request: RequestHeader)
 
+@import views.html.commercial.containerWrapper
+
```

Comment:
It was imported at the top of the file but I've moved it down now to make it more consistent.